src/app/components/BibPage/BibPage.jsx

```diff
@@ -32,11 +32,11 @@ const BibPage = (props) => {
   return (
     <DocumentTitle title={`${title} | Research Catalog`}>
       <main className="main-page">
-        <div className="nypl-page-header">
+        <div className="nypl-homepage-hero">
```

Comment:
@ricardoom, I added this class just for consistency but maybe the class name should be different?